app/views/diary_entry/view.html.erb

```diff
@@ -21,6 +21,11 @@
     <%= richtext_area :diary_comment, :body, :cols => 80, :rows => 15 %>
     <%= submit_tag t('diary_entry.view.save_button') %>
   <% end %>
+  <% if @user and @entry.subscribers and @entry.subscribers.exists?(@user.id) %>
```

Comment:
There isn't really any need to check `@entry.subscribers` here as it will always be non-nil as it is an association so will be an association proxy object.